src/custom/components/Popups/FollowPendingTxPopup/index.tsx

```diff
@@ -9,38 +9,95 @@ import {
   handleFollowPendingTxPopupAtom,
   handleHidePopupPermanentlyAtom,
   showFollowTxPopupAtom,
+  followPendingTxPopupAtom,
+  selectAtom,
+  handleCloseOrderPopupAtom,
 } from 'state/application/atoms'
 import FollowPendingTxPopupUI from './FollowPendingTxPopupUI'
 import { isPending } from 'components/Web3Status'
+import { OrderID } from '@src/custom/api/gnosisProtocol'
 
-function isAnOrder(element: TransactionAndOrder): element is Order & { addedTime: number } {
+interface AddedOrder extends Order {
+  addedTime: number
+}
+
+function isAnOrder(element: TransactionAndOrder): element is AddedOrder {
   return 'inputToken' in element && 'outputToken' in element
 }
```

Comment:
I think it makes more sense to define this type where it comes from, as well as the helper function

https://github.com/cowprotocol/cowswap/blob/b2525e609e197dbb937decb0c572a75b7ac187ea/src/custom/hooks/useRecentActivity.ts#L12